sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -308,12 +323,18 @@ aditof::Status CameraFxTof1::setFrameType(const std::string &frameType) {
     }
 
     if (m_details.frameType != *frameDetailsIt) {
+        uint16_t afeRegsAddr[2] = {0x4001, 0x7c22};
+        uint16_t afeRegsVal[2] = {0x0006, 0x0004};
+        m_depthSensor->writeAfeRegisters(afeRegsAddr, afeRegsVal, 2);
```

Comment:
What is happening here? Why is this required?

Reply:
We need to turn off the streaming before setting the frame type and then turning it back on.

Reply:
This needs to be documented next to the code.

Reply:
Fixed.